koji/vars/external_repos.yml

```diff
@@ -20,12 +20,6 @@ external_repos:
     url: file:///mnt/koji/external-repos/rhscl-rhel7-$arch/1.1/
   - name: epel-6
     url: file:///mnt/koji/external-repos/www/epel6-$arch/RPMS.epel/
-  - name: jpackage-5.0-generic-free
-    url: file:///mnt/koji/external-repos/www/jpackage-$arch/RPMS.jpackage/
-  - name: rhel-5.9-server
-    url: file:///mnt/koji/external-repos/rhel5-$arch/5.9/
-  - name: epel-5
-    url: file:///mnt/koji/external-repos/www/epel5-$arch/RPMS.epel/
```

Comment:
These are still used here:
https://github.com/theforeman/tool_belt/blob/04c8e7263df6720bb7482adb6f7bf06e9281cb23/configs/foreman/nightly.yaml#L365-L381

Reply:
Silly nightly, I'll go work on cleaning up nightly. We have not released this in the client repos since 2.2.

Reply:
PR to drop configs -- https://github.com/theforeman/tool_belt/pull/372

Reply:
Cleanup of nightly tag merged